app/src/main/java/org/simple/clinic/textInputdatepicker/TextInputDatePickerSheet.kt

```diff
@@ -20,6 +23,21 @@ class TextInputDatePickerSheet : BaseBottomSheet<
 
   override fun bindView(inflater: LayoutInflater, container: ViewGroup?) = SheetTextInputDatePickerBinding.inflate(inflater, container, false)
 
+  override fun events() = sheetCloseClicks()
+      .compose(ReportAnalyticsEvents())
+      .cast<TextInputDatePickerEvent>()
+
+  private val imageTextInputSheetClose
```

Comment:
Can we move these view binding variable declarations above the function calls? Currently, these are in between function calls.